incubator/monochart/templates/daemonset.yaml

```diff
@@ -41,18 +41,18 @@ spec:
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
 {{ include "monochart.env" . | indent 8 }}
-        args: {}
+        args: {{ .Values.daemonset.pod.args }}
         ports:
           - name: http
             containerPort: {{ .Values.service.internalPort }}
             protocol: TCP
         volumeMounts:
         - mountPath: /data
           name: storage
-        - mountPath: /config
-          name: config
-        - mountPath: /secret
-          name: secret
+        - mountPath: {{ .Values.configMap.mountPath }}
+          name: {{ .Values.configMap.volumeName }}
```

Comment:
Config map `volumeName` is should not be configurable

Reply:
why?

Reply:
Why we need this?
I can not imaging any use case for that